docs/pages/choose-an-edition/introduction.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "How to Choose a Teleport Edition"
+description: "Read this guide to determine whether to deploy Teleport Enterprise, Teleport Cloud, or Open Source Teleport"
+---
+
+Teleport supports three editions. In this guide, we will help you determine
+which edition is most appropriate for your use case.
+
+## Editions
+
+### Open Source Teleport
```

Comment:
I much prefer `Open Source Teleport`, but I think @xinding33 is starting to say "Teleport Community Edition" more nowadays. Let's confirm with him the naming here.

Reply:
The Core Concepts page says "Teleport Community Edition", but I must have let my muscle memory get in the way here—I'll change it